Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -1196,8 +1196,8 @@ extension System {
 
                 manager.query.getTransactions (blockchainId: manager.network.uids,
                                                addresses: addresses,
-                                               begBlockNumber: begBlockNumber,
-                                               endBlockNumber: endBlockNumber,
+                                               begBlockNumber: (begBlockNumber == BLOCK_HEIGHT_UNBOUND ? nil : begBlockNumber),
```

Comment:
What does a nil begin block number map to, a scan from 0 presumably?

Reply:
I don't know how BDB responds to a non-existent 'start_block', probably errors.

Reply:
> > @EBGToo, looks good. I updated the Java. I changed the Java wallet sweeper to use the `null` start/end height (rather than 0/`Network.height`). Does that seem like a reasonable change? Am I understanding correctly the BDB behaviour when the start number is omitted?
> 
> I don't know the BDB behavior if `start_height` is omitted. I made both 'start' and 'end' optional for uniformity reasons.

Okay, I walked back that change. Thanks!